tests/onnxruntime/utils_onnxruntime_tests.py

```diff
@@ -149,6 +149,7 @@
     "wav2vec2-conformer": "hf-internal-testing/tiny-random-wav2vec2-conformer",
     "wavlm": "hf-internal-testing/tiny-random-WavlmModel",
     "xlm": "hf-internal-testing/tiny-random-XLMModel",
+    "xlm_qa": "hf-internal-testing/tiny-random-XLMForQuestionAnsweringSimple",  # issue with default hf-internal-testing in transformers QA pipeline post-processing
```

Comment:
Thank you, `hf-internal-testing/tiny-random-XLMModel` is still above, is it intended?

Reply:
Yes `hf-internal-testing/tiny-random-XLMForQuestionAnsweringSimple` is only use for the QA tests, I think it make sense to keep `hf-internal-testing/tiny-random-XLMModel` for other tasks and tests

Reply:
Oh I misread, I see.